app/controllers/app_controller.rb

```diff
@@ -13,8 +13,14 @@ class TodoApp < Sinatra::Base
     erb :'home/index', :layout => :'layouts/layout'
   end
 
+  # get '/all-tasks/:id/:name' do 
+  #   puts params
+  # end
+
   get '/all-tasks' do
     @tasks = JSON.parse(Task.get_all_tasks)
+    puts @tasks
```

Comment:
I'm guessing these  puts statements were here to help with debugging? Probably also worth trying to remove these prior to opening a pr.

Reply:
👍